### ISSUE TITLE
Fix dev build

### ISSUE THREAD

**Issue by kwalrath** [dart-lang/site-www#1663]
More info: #1662

**Comment by kwalrath** [comment]
As of 2.3.2-dev.0.1, code that was passing in previous dev releases is now failing. From https://travis-ci.org/dart-lang/site-www/jobs/541318807:

```
Analyzing lib, test...
  hint • This function has a return type of 'bool', but doesn't end with a return statement at lib/effective_dart/docs_good.dart:6:3 • missing_return
  hint • This function has a return type of 'String', but doesn't end with a return statement at lib/effective_dart/usage_bad.dart:45:3 • missing_return
  hint • This function has a return type of 'String', but doesn't end with a return statement at lib/effective_dart/usage_good.dart:53:3 • missing_return
3 hints found.
No analysis errors or warnings should be present in original source files.
Ensure that these issues are disabled using appropriate markers like: 
  // ignore_for_file: 2, some_analyzer_error_or_warning_id
```

I've temporarily disabled dev testing, and now I'm going to take a look at whether this failure is a good thing.

/cc @johnpryan @bwilkerson

**Comment by kwalrath** [comment]
We can't fix this until we're ready to cut over to 2.3.2 — presumably that'll be when 2.3.2 is stable.

**Comment by kwalrath** [comment]
It's working now.
